src/components/calendar/calendar.tsx

```diff
@@ -338,7 +338,14 @@ export const Calendar: React.FC<CalendarProps> = ({
         x={0}
         y={0}
         width={columnWidth * dateSetup.dates.length}
-        height={headerHeight}
+        height={45}
```

Comment:
is height fixed or are we passing it down from parent component? Where we set the row height?

If we're setting it from parent, I would suggest to not hardcode values here and instead use the variable value from parent.